### ISSUE TITLE
Add chromium launch options

### ISSUE THREAD

**Issue by gustavomassa** [jsreport/jsreport-chrome-pdf#17]
Hello,

I'm having a lot of issues regarding chromium on CentOS 7:
https://github.com/GoogleChrome/puppeteer/issues/391

I'm facing this error:
`pupeteer Running as root without --no-sandbox is not supported`

So I'm installing pupeteer manually like this:
```
cd /home/chromium
git clone https://github.com/scheib/chromium-latest-linux .
./update.sh

cd /home/chromium/latest
chown yourID:yourID *
# Need root permission
chown root:root chrome_sandbox && chmod 4755 chrome_sandbox
export CHROME_DEVEL_SANDBOX="/home/chromium/latest/chrome_sandbox"

export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
npm install puppeteer
```
Now I need the launch options to pass the executablePath:
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md

Something passed directly to jsreport-chrome-pdf:
```
jsreport.use(require('jsreport-chrome-pdf')(
{
 executablePath: './home/chromium-linux/latest',
 args: ['lala', 'lele']
}
));
```
The arg '--no -sandbox' should resolve the issue.
Please let me know if exists any other ways of resolving this issue.

Thanks!

**Comment by bjrmatos** [comment]
hi! chrome's launch options are already supported, just read the relevant section in the [docs](https://jsreport.net/learn/chrome-pdf#configuration).

TL;DR

```js
jsreport.use(require('jsreport-chrome-pdf')(
{
  launchOptions: {
    executablePath: './home/chromium-linux/latest',
    args: ['lala', 'lele'] // or ['--no-sandbox']
  }
}
));
```

**Comment by gustavomassa** [comment]
Adding --no-sandbox and executablePath resolved the issue.
Thank you for pointing the doc.
Could you please update the https://github.com/jsreport/jsreport-chrome-pdf/blob/master/README.md readme with that doc info?

**Comment by bjrmatos** [comment]
in some repositories (like this one) and with our current docs setup we avoid duplicating the options, instead we link to online docs to avoid updating two places when we have the need. maybe in future we will be able to show the same information in both places from single source

**Comment by pofider** [comment]
Also linking the full centos 7 installation instructions
https://jsreport.net/learn/centos
This and other links can be found on the download page
https://jsreport.net/on-prem

**Comment by jayaprakash369** [comment]
> hi! chrome's launch options are already supported, just read the relevant section in the [docs](https://jsreport.net/learn/chrome-pdf#configuration).
> 
> TL;DR
> 
> ```js
> jsreport.use(require('jsreport-chrome-pdf')(
> {
>   launchOptions: {
>     executablePath: './home/chromium-linux/latest',
>     args: ['lala', 'lele'] // or ['--no-sandbox']
>   }
> }
> ));
> ```

may I know the path for this config file?

**Comment by pofider** [comment]
@whitedevil-369 https://jsreport.net/learn/chrome-pdf#configuration
